django_pony_express/services/base.py

```diff
@@ -109,6 +109,7 @@ class BaseEmailService:
     """
 
     SUBJECT_PREFIX = None
+    SUBJECT_DELIMITER = " - "
```

Comment:
nitpick/issue: this is hard to keep in mind. i would argue that spaces are to be expected around a delimiter so I'd leave them out here and put them in the string.

Reply:
valid.

I wanted to keep it in, because it seems to me, as if you were following some kind of convention, when you created your default delimiter " - ".

Since the people that use django-pony-express so far, have seemingly decided, that this works for them, I wanted to keep the delimiter as untouched as possible.

Including the spaces in the delimiter however, gives them the opportunity to create delimiters such as ": ", which I thought was a nice touch.

Otherwise you're forcing developers to use delimiters surrounded by spaces.

What do you think?

Reply:
That's a valid point. Should we add this to the docs? 🤔

Reply:
Wouldn't you say that [this line](https://github.com/ambient-innovation/django-pony-express/pull/19/files#diff-aab919b2a60fe48bbc09f896c564cae93ea31709ddbe34e6a811015ee15a05a5R6) does it?